imgparse/imgparse.py

```diff
@@ -83,6 +83,15 @@ def get_timestamp(image_path, exif_data=None, format_string="%Y:%m:%d %H:%M:%S")
     :return: **datetime_obj**: Parsed timestamp, in the format specified by the input format string.
     :raises: ParsingError
     """
+    try:
+        from timezonefinder import TimezoneFinder
+    except ImportError:
+        logger.warning(
+            "Module timezonefinder is required for retrieving timestamps."
+            "Please execute `poetry install -E dji_timestamps` to install this module."
```

Comment:
Let's rename the extras requires to just `timestamps` since this applies to all sensors now

Reply:
Addressed this myself